app/workers/feeds/import_articles_worker.rb

```diff
@@ -18,7 +19,27 @@ def perform(user_ids = [], earlier_than = nil)
         earlier_than ||= 4.hours.ago
       end
 
-      ::Feeds::Import.call(users_scope: users_scope, earlier_than: earlier_than)
+      # For some reason `ActiveSupport::TimeWithZone#is_a?(Time)` evaluates to
+      # `true` so this works with any sort of time object
+      if earlier_than.is_a?(Time)
+        earlier_than = earlier_than.iso8601
+      end
+
+      users_scope.select(:id).find_in_batches do |batch|
+        ids = batch.map { |user| [user.id, earlier_than] }
+
+        ForUser.perform_bulk(ids)
```

Comment:
I was surprised by the var name `ids`.  It's an array of the `id` and `earlier_than`.

Perhaps `params`.  I'm also stumbling with method signatures.  The `perform` below is `user_ids` and `earlier_than`.  Which looks off based on what `ids` is in the above map.

Reply:
@jeremyf Good catch! This was originally just [an array containing the `user.id` args](https://github.com/forem/forem/pull/16818/commits/33574637e7dc9d5b99b67a83bac94a9600ee6a9f#diff-1949504a3295ae8dec7f764285907d8b067acc66ae56a873e6280ca7b34028faR23) (which didn't work because I forgot how `perform_bulk` works in Sidekiq), but even that was still an array of argument lists, not technically ids, so it wasn't an accurate name even then.

Reply:
@jeremyf Renamed to `arg_lists`. `params` has other connotations in Rails, most notably that it is typically hash-like and Sidekiq requires arrays.